### ISSUE TITLE
No trace no more

### ISSUE THREAD

**Issue by arichiardi** [day8/re-frame-tracer#2]
I was using this library with success in `lein`, switching to `boot` muted the tracing.
I am not even sure that the `lein` to `boot` passage is related, but basically after a bit of debugging I see that `js/clairvoyant.core.devmode` is always false no matter what `:elide-asserts` is.

I wanted to dig more and I migth be blind but I cannot really find where `clairvoyant.core/devmode` is defined in the code source.

I need some little hint :smile:

**Comment by arichiardi** [comment]
By the way I found a workaround, and I don't think `boot` is related, there must be some other problem with the google define you have in your code (line 80 of `core.clj`, I went through my `.m2`):

I added: `:closure-defines {"clairvoyant.core.devmode" true}`

**Comment by danielcompton** [comment]
That's not a workaround, that's expected behaviour :). You need to provide a closure-define to enable/disable it IIRC.

**Comment by arichiardi** [comment]
Oh, but the pure `clairvoyant` is supposed to enable/disable itself on `:elide-assert` that's why I reported it...Maybe we just need a note in the README?

**Comment by mike-thompson-day8** [comment]
@arichiardi we provided the original patch to clairvoyant so it would use `:elide-assert` but then we backed away from that method and suggested using goog defines instead (it was a new feature at that time).  But our patch appears stuck,  which is one reason why we have our own fork
https://github.com/spellhouse/clairvoyant/pull/14

But the bottom line is that our fork is subtly different on that point.  And, sorry,  you are right that a note in the README would be a good idea.
